src/main/java/com/microsoft/sqlserver/jdbc/osgi/SQLServerDataSourceFactory.java

```diff
@@ -22,15 +24,16 @@
 
 
 /**
- * Implementation of the Data Service Specification for JDBC™ Technology
- * 
- * see https://osgi.org/specification/osgi.cmpn/7.0.0/service.jdbc.html
+ * Implementation of the Data Service Specification for JDBC™ Technology. Refer to the OSGI 7.0.0 specifications for
+ * more details.
  */
 public class SQLServerDataSourceFactory implements DataSourceFactory {
 
     private static java.util.logging.Logger osgiLogger = java.util.logging.Logger
             .getLogger("com.microsoft.sqlserver.jdbc.osgi.SQLServerDataSourceFactory");
-    private static String NOT_SUPPORTED_MSG = "The Microsoft SQL Server JDBC Driver currently does not support the property: {0}";
+    private static final String NOT_SUPPORTED_MSG = ResourceBundle
+            .getBundle("com.microsoft.sqlserver.jdbc.SQLServerResource", Locale.getDefault())
+            .getString("R_propertyNotSupported");
```

Comment:
You could simply call `SQLServerResource.getResource("R_propertyNotSupported")`.